RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,24 +8,45 @@
 
 import Cocoa
 
+@objc protocol EntitiesViewDataSource: NSObjectProtocol {
+    optional func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    optional func entitiesView(entitiesView:EntitiesView, titleForEntityAtIndex index:Int) -> String?
+}
+
+@objc protocol EntitiesViewDelegate: NSObjectProtocol {
+    optional func addEntityInEntitiesView(entitiesView:EntitiesView)
+    optional func removeEntityInEntitiesView(entitiesView:EntitiesView, index:Int)
```

Comment:
function at line 18 and 19 are technically the same function. remove the one on line 18 and just he the one on line 19